src/main/src/test/java/org/geoserver/test/GeoServerSystemTestSupport.java

```diff
@@ -164,9 +165,33 @@ protected SystemTestData createTestData() throws Exception {
     /**
      * credentials for mock requests
      */
-    protected String username, password; 
+    protected String username, password;
+
+    /**
+     * Cached dispatcher, it has its own app context, so it's expensive to build
+     */
+    protected static DispatcherServlet dispatcher;
+
+    protected boolean isFullInitRequired() {
+        return false;
+    }
 
     protected final void setUp(SystemTestData testData) throws Exception {
+        // speed up xpath evaluations
+        try {
+            // see
+            // http://stackoverflow.com/questions/6340802/java-xpath-apache-jaxp-implementation-performance
+            Class.forName("com.sun.org.apache.xml.internal.dtm.ref.DTMManagerDefault");
```

Comment:
This speeds up xpath evaluations, we are doing a number of them. The Class.forName should be guarding against JDK that do not have the required classes.